scripts/test_endpoints.py

```diff
@@ -96,5 +98,15 @@ def pixel_history(x: int, y: int) -> None:
     a.raise_for_status()
 
 
+@cli.command()
+def get_size() -> None:
+    """Gets the size of the canvas using no auth."""
+    a = requests.get(
+        f"{base_url}/size"
+    )
```

Comment:
I understand that this _may_ have been done to be similar to the other tests
```suggestion
    a = requests.get(f"{base_url}/size")
```